packages/docusaurus-theme-classic/src/theme/Navbar/styles.module.css

```diff
@@ -16,5 +16,5 @@
 }
 
 .navbarHidden {
-  transform: translate3d(0, calc(var(--ifm-navbar-height) * -1), 0);
+  transform: translate3d(0, -103%, 0);
```

Comment:
Since the navbar uses box shadow, we need to hide it as well.

![2021-06-14_09-44](https://user-images.githubusercontent.com/4408379/121850626-b6bfa980-ccf5-11eb-81b0-97ba0bc4dbfb.png)

Reply:
Shouldn't we add the box shadow value in calc then? Not 100% comfortable having a 3% magic value here, there's no change I'll understand why it's here in 1 year. 

If we keep such value, can you add a comment, make it part of calc and use a px value instead of 3%? as far as I understand the shadow is 2px?

Reply:
It is possible, but I didn't want to apply complicated/long calculations. So I decided to use the percentage value, I can add just comment, is that okay with you?

Reply:
What about `calc(-100% - 2px)` and a comment to say 2px is the shadow?

Reply:
Right, did so.